modules/nw-secondary-ext-gw-object.adoc

```diff
@@ -37,6 +32,8 @@ from:
       kubernetes.io/metadata.name: novxlan-externalgw-ecmp-4059
 ----
 
+A namespace can be targeted by only one `AdminPolicyBasedExternalRoute` CR. If a namespace is selected by more than one `AdminPolicyBasedExternalRoute` CR, a `failed` error status occurs.
```

Comment:
may I suggest: a `failed` error status occurs on the second and subsequent CRs targeting the same namespace.

Just to clarify that the first CR will be successful regardless of how many additional CRs are added later on.